### PR TITLE
Feature/net management docs

### PR DESCRIPTION
**Pull request by Roblinde**

### CODE REVIEW

File: docs/net/tutorials/managment-api.md [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+~~~csharp
+var contentType = new ContentType();
+            contentType.SystemProperties = new SystemProperties();
+            contentType.SystemProperties.Id = "<content_type_id>"; //Specify an identifier, must be between 2 and 60 characters long and may only contain alphanumerical characters and dash(-), underscore(_) or period(.).
+            contentType.Name = "ContentTypeTest";
+            contentType.Description = "Description";
+            contentType.DisplayField = "field1";
+            contentType.Fields = new List<Field>()
+            {
+                new Field()
```
Comment (ChrisChinchilla) [inline docs/net/tutorials/managment-api.md]: @Roblinde Can I clarify what you're trying to create here? A list / array field?

Comment (Roblinde) [inline docs/net/tutorials/managment-api.md]: Just a regular field of type Symbol with a validation of a length between 2 and 8 characters. Or am I misunderstanding you?

**Comment by ChrisChinchilla** [comment]
@Roblinde I've finished my review of _docs/net/tutorials/managment-api.md_, so take a look at let me know what you think.

File: docs/net/tutorials/managment-api.md [hunk 2]
```diff
@@ -45,192 +44,192 @@ var space = await client.CreateSpaceAsync("<space_name>", "<default_locale>", "<
 Console.WriteLine(space.Name); // => <space_name>
 ```
 
-If your user account belongs to a single organization the organization_id parameter can be omitted.
+If your user account belongs to a single organization, you can omit the `organization_id` parameter.
```
Comment (Roblinde) [inline docs/net/tutorials/managment-api.md]: , needed?

File: docs/net/tutorials/managment-api.md [hunk 3]
```diff
@@ -45,192 +44,192 @@ var space = await client.CreateSpaceAsync("<space_name>", "<default_locale>", "<
...
 
 To delete a space, pass a space id to the `DeleteSpaceAsync` method:
 
 ```csharp
 var space = await client.DeleteSpaceAsync("<space_id>");
 ```
 
-To change the name of an existing space use the `UpdateSpaceNameAsync` method.
+To change the name of an existing space, use the `UpdateSpaceNameAsync` method.
```
Comment (Roblinde) [inline docs/net/tutorials/managment-api.md]: , needed?

File: docs/net/tutorials/managment-api.md [hunk 4]
```diff
@@ -45,192 +44,192 @@ var space = await client.CreateSpaceAsync("<space_name>", "<default_locale>", "<
...
 
 ```csharp
 var space = await client.UpdateSpaceNameAsync("<space_id>", "<new_space_name>", "<space_version>", "<organization_id>");
 ```
 
-The organisation id can, again, normally be omitted unless your account has several different organizations. The version parameter however, can not.
+Unless your account has more than one organization, you can omit the organization id, but the version parameter is always needed.
```
Comment (Roblinde) [inline docs/net/tutorials/managment-api.md]: first , needed?

File: docs/net/tutorials/managment-api.md [hunk 5]
```diff
@@ -45,192 +44,192 @@ var space = await client.CreateSpaceAsync("<space_name>", "<default_locale>", "<
...
 
-This is a common pattern to update operations in the Contentful management API. To update an entry you need to pass the last known version to make sure that you do not overwrite a resource that has since been updated. This is known as optimistic locking and is to prevent unwanted data loss. If the version passed does not match the latest version in Contentful the update will be rejected and a `ContentfulException` will be thrown.
+This is a common pattern to update operations in the Contentful management API. To update an entry you need to pass the last known version to make sure that you do not overwrite a resource that has since been updated. This is called 'optimistic locking' and prevents unwanted data loss. If the version passed does not match the latest version in Contentful the update will be rejected and a `ContentfulException` thrown.
 
-To retrieve the version of a resource we inspect the `SystemProperties.Version` property.
+To retrieve the version of a resource, inspect the `SystemProperties.Version` property.
```
Comment (Roblinde) [inline docs/net/tutorials/managment-api.md]: This , actually feels right for some reason unknown to me... but might as well review it 😄

File: docs/net/tutorials/managment-api.md [hunk 6]
```diff
@@ -45,192 +44,192 @@ var space = await client.CreateSpaceAsync("<space_name>", "<default_locale>", "<
...
 var space = await client.GetSpaceAsync("<space_id>")
-var version = space.SystemProperties.Version; //Nullable int
+var version = space.SystemProperties.Version; // Nullable int
 await client.UpdateSpaceNameAsync("<space_id>", "<new_space_name>", version.Value);
 ```
 
 ## Working with content types
 
-Once you've familiarized yourself with creating and deleting spaces the next step is to add some content types to your space. A content type in Contentful is a blue print for an Entry, it contains up to 50 fields that you can define.
+Once you've familiarized yourself with creating and deleting spaces the next step is to add content types to your space. A content type in Contentful is a blue print for an entry, and contains up to 50 fields that you can define.
```
Comment (Roblinde) [inline docs/net/tutorials/managment-api.md]: Is it blue print or blueprint?

File: docs/net/tutorials/managment-api.md [hunk 7]
```diff
@@ -242,37 +241,37 @@ The fields have a lot of properties and can look daunting at first, especially i
...
+    LinkType = "Entry",  // Applicable if Type is "Link", whether this links to an Entry or an Asset   
+    Items = new Schema() {  // Applicable if the Type is "Array" and specifies allowed item types for the array
+        Type = "Link", // The type this array field should contain, e.g. "Symbol" or "Link"
+        LinkType = "Entry" // Applicable if the item type is Link, specifies whether the array contains Entry or Asset
+    },
+    Validations = new List<IFieldValidator>() // See validations section below                
+}
 ```
 
 But at minimum, you need to specify the name, id and type.
```
Comment (Roblinde) [inline docs/net/tutorials/managment-api.md]: , needed?

Comment (Roblinde) [inline docs/net/tutorials/managment-api.md]: , needed?

**Comment by ChrisChinchilla** [comment]
@Roblinde And webhooks doc also reviewed.

File: docs/net/tutorials/webhook-roles-snapshots.md [hunk 8]
```diff
@@ -12,95 +11,93 @@ nextsteps:
...
     "Entry.*"
 };
 
 await client.CreateWebHookAsync(webHook);
-~~~
+```
 
-The topics available are described in more detail in [the webhooks concepts article](/developers/docs/concepts/webhooks/), but the rundown is that you specify a Type and an Action,
-where type can be any of Entry, Asset or Content_type and Action can be any of create, save, auto_save, archive, unarchive, publish, unpublish or delete. * is a wildcard and *.* is valid
+The topics available are described in more detail in [the webhooks concepts article](/developers/docs/concepts/webhooks/), but the summary is that you specify a `Type` and an `Action`. `Type` can be any of `Entry`, `Asset` or `ContentType` and `Action` can be any of `create`, `save`, `auto_save`, `archive`, `unarchive`, `publish`, `unpublish` or `delete`. `_` is a wildcard and `_.\*` is valid
```
Comment (Roblinde) [inline docs/net/tutorials/webhook-roles-snapshots.md]: Should be `*` is a wildcard, not _

File: docs/net/tutorials/webhook-roles-snapshots.md [hunk 9]
```diff
@@ -118,110 +115,106 @@ role.Policies.Add(new Policy() //Every policy consists of a number of actions an
...
-The policies can look quite daunting at first look and the framework behind them is quite complex. It does, however, make it possible to create highly granular and flexible authorization rules.
+The policies can look daunting at first and the framework behind them is complex. However they do make it possible to create granular and flexible authorization rules.
 
 The first property is the `Effect` which is how this policy is to be treated, will it `allow` or `deny` access.
 
-Then comes `Actions` which represent what actions this policy allow or deny. This is a list of strings but only a certain set of values are acceptable. "read", "create", "update", "delete", "publish", "unpublish", "archive", "unarchive" or "all".
+`Actions` represents what actions this policy allows or denies. This is a list of strings but only a certain set of values are acceptable. These are `read`, `create`, `update`, `delete`, `publish`, `unpublish`, `archive`, `unarchive` or `all`.
 
-Then we have the `Constraint` which is a `IConstraint` explained in more detail below.
+`Constraint` is a `IConstraint`, which is explained in more detail below.
```
Comment (Roblinde) [inline docs/net/tutorials/webhook-roles-snapshots.md]: should that be "an IConstraint"?

File: docs/net/tutorials/webhook-roles-snapshots.md [hunk 10]
```diff
@@ -239,25 +232,28 @@ newMembership.User = new User() { //The user must already exist. It cannot be au
...
-~~~csharp
+```csharp
 await client.DeleteSpaceMembershipAsync("<membership_id>");
-~~~
+```
 
 ## Working with API keys
 
-The Contentful .NET SDK allows you to get all API keys for the CDA (Content Delivery API, not the managament API) for a space. It also allows you to create new API keys as needed.
+The Contentful .NET SDK allows you to get all API keys for the Content Delivery API ((not the management API) for a space. It also allows you to create new API keys.
```
Comment (Roblinde) [inline docs/net/tutorials/webhook-roles-snapshots.md]: an extra ( snuck in.

File: docs/net/tutorials/managment-api.md (hunk 2 above)
Comment (ChrisChinchilla) [inline docs/net/tutorials/managment-api.md]: @Roblinde Run it through 2 checkers and both say OK…

**Pull request merged**